api/database/factories/UserFactory.php

```diff
@@ -294,20 +294,18 @@ public function asAdmin()
     /**
      * Attach the process operator role to a user after creation.
      *
-     * @param  string|array  $poolId  Id of the pool or pools to attach the role to
+     * @param  string|array  $team  Name of the team or teams to attach the role to
      * @return $this
      */
-    public function asProcessOperator(string|array $poolId)
+    public function asProcessOperator(string|array $team)
```

Comment:
Noooo, this was better before! 😭 

Just like `asCommunityRecruiter`, which takes in a communityId, this should take in a poolId.

Reply:
I prefer taking in a pool instead as well but the pool doesn't exist yet if I call this in UserTestSeeder.

Reply:
How about [this](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10621/commits/1e2ad77ea2956fe8c2441aa152a086371aeaf755) instead?